src/components/forms/DiskDeviceFormCustom.tsx

```diff
@@ -38,10 +42,18 @@ const DiskDeviceFormCustom: FC<Props> = ({
   const focusField = (name: string) => {
     setTimeout(() => document.getElementById(name)?.focus(), 100);
   };
+  const notify = useNotify();
+
+  const { data: profiles = [], error: profileError } = useQuery({
+    queryKey: [queryKeys.profiles],
+    queryFn: () => fetchProfiles(project),
+  });
+
+  if (profileError) {
+    notify.failure("Loading profiles failed", profileError);
+  }
```

Comment:
We can get the profiles from the parent, that seems the simpler approach than loading them again here.